ci-operator/step-registry/upi/install/heterogeneous/powervs/upi-install-heterogeneous-powervs-commands.sh

```diff
@@ -132,13 +123,9 @@ function cleanup_prior() {
             sleep 60
         fi
 
-        echo "Deleting the Network"
-        for NETWORK_ID in $(ibmcloud pi network ls 2> /dev/null | awk '{print $1}')
-        do
-        echo "Deleting network ${NETWORK_ID}"
-        ibmcloud pi network delete "${NETWORK_ID}" || true
-        sleep 60
-        done
+        echo "Delete network non-'ocp-net' on PowerVS region"
+        ibmcloud pi subnet ls --json | jq -r '[.networks[] | select(.name | contains("ocp-net") | not)] | .[]?.networkID' | xargs --no-run-if-empty -I {} ibmcloud pi subnet delete {} || true
+        echo "Done deleting non-'ocp-net' on PowerVS"
```

Comment:
This is a better version of the erase, and will ignore ocp-net if it's in the workspace